### PR TITLE
add tidb-cloud limited-sql for plan replayer

### PR DESCRIPTION
**Pull request by zeminzhou**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/limited-sql-features.md [hunk 1]
```diff
@@ -238,6 +238,6 @@ TiDB Cloud works with almost all workloads that TiDB supports, but there are som
 
 [^10]: The variable is read-only on TiDB Serverless.
 
-[^11]: Customizing cache size using [`AUTO_ID_CACHE`](/auto-increment.md#cache-size-control) is temporarily unavailable on TiDB Serverless.
+[^11]: TiDB Serverless does not support downloading the file through tidbstatus port as in the [example](/sql-plan-replayer.md#examples-of-exporting-cluster-information), but TiDB Serverless will generate a [presigned URL](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ShareObjectPreSignedURL.html) for users to download the file. It should be noted that the validity period of this URL is 10 hours.
```
Comment (baiyuqing) [inline tidb-cloud/limited-sql-features.md:241]: `example` is a dead link.

Comment (zeminzhou) [reply tidb-cloud/limited-sql-features.md]: fixed, thanks!

File: tidb-cloud/limited-sql-features.md [hunk 2]
```diff
@@ -237,6 +238,6 @@ TiDB Cloud works with almost all workloads that TiDB supports, but there are som
 
 [^10]: The variable is read-only on TiDB Serverless.
 
-[^11]: Customizing cache size using [`AUTO_ID_CACHE`](/auto-increment.md#cache-size-control) is temporarily unavailable on TiDB Serverless.
+[^11]: TiDB Serverless does not support downloading the file through tidbstatus port as in the [example](https://docs.pingcap.com/tidb/stable/sql-plan-replayer#examples-of-exporting-cluster-information), but TiDB Serverless will generate a [presigned URL](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ShareObjectPreSignedURL.html) for users to download the file. It should be noted that the validity period of this URL is 10 hours.
```
Comment (qiancai) [inline tidb-cloud/limited-sql-features.md:241]:
```suggestion
[^11]: TiDB Serverless does not support downloading the file exported by `PLAN REPLAYER` through `${tidb-server-status-port}` as in the [example](https://docs.pingcap.com/tidb/stable/sql-plan-replayer#examples-of-exporting-cluster-information). Instead, TiDB Serverless generates a [presigned URL](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ShareObjectPreSignedURL.html) for you to download the file. Note that this URL remains valid for 10 hours after generation.
```

**Review by ystaticy — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**